src/hooks/restrict-to-owner.js

```diff
@@ -0,0 +1,42 @@
+import errors from 'feathers-errors';
+
+const defaults = {
+  idField: '_id',
+  ownerField: 'userId'
+};
+
+export default function(options = {}){
+  return function(hook) {
+    if (hook.type !== 'before') {
```

Comment:
I would think this hook could also be an `after` hook for `get` where it just checks the `ownerField` of `result`